superset/views/log/views.py

```diff
@@ -24,3 +24,4 @@
 
 class LogModelView(LogMixin, SupersetModelView):  # pylint: disable=too-many-ancestors
     datamodel = SQLAInterface(models.Log)
+    include_route_methods = {"list"}
```

Comment:
Consider adding "show" also, since the fields retrieved by list are not very useful, it's important to access the log detail

Reply:
Unless this is 100% required for application functionality, I would recommend against exposing this information.

Reply:
I think what makes sense is either `{"list", "show"}` or nothing here. I was thinking about leaving it for backwards compatibility. I could put it behind a config flag too, let me do that.